docs/module/dhcp.md

```diff
@@ -56,17 +56,21 @@ DHCP relays are supported on these platforms:
 The DHCP module supports the following node parameters:
 
 * **dhcp.server** (boolean) -- the device should run a DHCP server
-* **dhcp.vrf** (boolean) -- the DHCP relay or server should be VRF-aware. The default value of this parameter is set to _true_ if the node performs inter-VRF DHCP relaying or has DHCP pools from VRF subnets. To override that behavior, you can set **dhcp.vrf** to _false_.
+* **dhcp.vrf** (boolean) -- the DHCP relay or server should be VRF-aware. The default value of this parameter is set to _true_ if the node performs inter-VRF DHCP relaying or has DHCP pools from VRF subnets. You can override that behavior by setting **dhcp.vrf** to _false_.
 
 ```{tip}
 You have to set **‌dhcp.vrf** to _False_ on a Cisco IOS XE DHCP server when you use it with Arista EOS VRF-aware relays ([more details](https://blog.ipspace.net/2023/03/netlab-vrf-dhcp-relay.html)).
 ```
- 
+
+## Link Parameters
+
+You can enable the DHCP client on all non-relaying devices attached to a link with the boolean **dhcp.client.ipv4** link attribute and the DHCPv6 client with the boolean **dhcp.client.ipv6** attribute.
+
 ## Interface Parameters
 
-* To start a DHCP client on an interface, set **ipv4** or **ipv6** address to **dhcp**.
-* The DHCP client usually installs a default route to an IPv4 default gateway[^DGv6] sent in a DHCP reply. To disable this functionality, set **dhcp.client.default** interface parameter to `False`.
-* To start a DHCP relay on an interface, set **dhcp.server** interface parameter to a list of DHCP servers.
+* To start a DHCP client on an interface, set the **ipv4** or **ipv6** address to **dhcp**. You can use the **dhcp.client.ipv4** or the **dhcp.client.ipv6** interface attribute only when the node already uses the DHCP module. 
+* The DHCP client usually installs a default route to an IPv4 default gateway[^DGv6] sent in a DHCP reply. To turn off this functionality, set **dhcp.client.default** interface parameter to `False`.
```

Comment:
Shouldn't this be ```dhcp.client.ipv4.default```?

Reply:
If we set **dhcp.client.ipv4** to **True**, how can we then set **dhcp.client.ipv4.default** to something?

Reply:
Various options:
1. dhcp.client.ipv4:
      request: True
      default: False
2. dhcp.client.ipv4_default: False

It's just to emphasize that this applies to ipv4 only

Reply:
DHCPv6 does not have a "default gateway" option. IPv6 uses RA to get the default gateway information.